utils.groovy

```diff
@@ -217,6 +227,28 @@ def bump_builds_json(stream, buildid, arch, s3_stream_dir, acl) {
     }
 }
 
+// Make a COSA remote session, which is usually used to
+// build on a different architecture.
+//
+// Available parameters:
+//    env: (optional) map of key/val pairs of environment variables to set
+//    expiration: string that represents a golang duration for how
+//                long the container should last (i.e. 4h, 30m)
+//    image: string that represents the container image to pull
+//    workdir: string that represents the in container working directory
+def makeCosaRemoteSession(params = [:]) {
+    def env = params['env'] ?: [:]
```

Comment:
Though note this is shadowing the builtin `env` var. I don't think it matters in this case, but would be cleaner to not shadow.

Reply:
fixed

Reply:
reworked things to not shadow the `env` var